internal/ignition/ignition.go

```diff
@@ -729,14 +768,18 @@ func (g *installerGenerator) updateIgnitions() error {
 			return errors.Wrapf(err, "error adding CA cert to ignition %s", workerPath)
 		}
 	}
-	if len(g.cluster.Hosts) >= 1 {
-		ipv6Only, err := network.AreIpv6OnlyHosts(g.cluster.Hosts, g.log)
-		if err != nil {
-			return err
-		}
-		if ipv6Only {
-			for _, ignition := range []string{"master.ign", "worker.ign"} {
-				if err = g.addIpv6FileInIgnition(ignition); err != nil {
+
+	ipv6Only, err := network.AreIpv6OnlyHosts(g.cluster.Hosts, g.log)
+	if err != nil {
+		return err
+	}
+	if ipv6Only {
+		for _, ignition := range []string{masterIgn, workerIgn} {
+			if err = g.addIpv6FileInIgnition(ignition); err != nil {
+				return err
+			}
+			if g.cluster.ImageInfo.StaticIpsConfig != "" {
```

Comment:
Will it hurt if we do it for static IPs regardless of IPv6 only, outside the `if`?

Reply:
no, it won't. I am not moving files, i am coping them, so they will be in the default path also. I have set it here so that the reason will be clear. Once/if Ori will move ipv6 confd to be always present, we will move this code also